packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -210,8 +218,31 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
   async _verifySignature(
     commitData: CommitData,
     context: Context,
-    controller: string
+    controller: string,
+    streamId?: StreamID
   ): Promise<void> {
+    if (streamId) {
```

Comment:
Why is this conditional on `streamId`? Should it not be conditional on if there is a `cap` prop in the JWS header?

Reply:
The only time `streamid` is passed is when applying a signed commit. It then only does the decoding for protected headers if we are trying to verify a signed commit

Reply:
nevermind, i see genesis blocks can also be signed. moved it.